### PR TITLE
Makefile: Add check-fedora-versions-sys target

### PR DESCRIPTION
**Pull request by bgurney-rh**
Related stratis-storage/project#307

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -19,6 +19,11 @@ build-deprecated:
 check-fedora-versions:
 	`${COMPARE_FEDORA_VERSIONS} | jq '[.missing == ["libblkid-rs-sys"], .high == []] | all'`
 
+check-fedora-versions-sys:
+	`${COMPARE_FEDORA_VERSIONS} \
+	--manifest-path=./libblkid-rs-sys/Cargo.toml \
+	| jq '[.missing == ["libblkid-rs-sys"], .high == []] | all'`
```
Comment (mulkieran) [inline Makefile:25]: Bet you can drop "libblkid-rs-sys" in this line. It ought not to depend on itself.

**Review by mulkieran — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (bgurney-rh) [reply Makefile]: Confirmed; it works after removing it.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Ok! Next trick is to figure out how to invoke it from GitHub Actions. The trick is that the working directory is that specified by the GitHub Actions configuration, not where the Makefile is located.

**Review by mulkieran — CHANGES_REQUESTED** [review]
Looks good! Please close this and open a new PR for final review.

**Pull request closed without merging**